test/datasets/speechcommands_test.py

```diff
@@ -0,0 +1,73 @@
+import os
+import random
+
+from torchaudio.datasets import speechcommands
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    normalize_wav,
+    save_wav,
+)
+
+
+class TestSpeechCommands(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    samples = []
+
+    @classmethod
+    def setUp(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        dataset_dir = os.path.join(
+            cls.root_dir, speechcommands.FOLDER_IN_ARCHIVE, speechcommands.URL
+        )
+        os.makedirs(dataset_dir, exist_ok=True)
+        sample_rate = 16000  # 16kHz sample rate
+        random.seed(0)
```

Comment:
Please do not use random in test. It adds complexity without benefit. See my comment https://github.com/pytorch/audio/pull/827#discussion_r460899418 for why.

Reply:
Simply use integer value as seed and increment it.